app/models/agents/website_agent.rb

```diff
@@ -343,7 +343,7 @@ def receive(incoming_events)
               if url_template = options['url_from_event'].presence
```

Comment:
The problem is the other direction: `url_from_event` and `data_from_event` are both blank.  I guess in this case, if the Agent is wired to receive events, those events currently *must* have a `url` key for anything to be working, so we could set the Agent's `url_from_event` to `{{ url }}`?